postgres-appliance/configure_spilo.py

```diff
@@ -251,6 +251,7 @@ def get_placeholders(provider):
     placeholders.setdefault('BACKUP_SCHEDULE', '00 01 * * *')
     placeholders.setdefault('CRONTAB', '[]')
     placeholders.setdefault('PGROOT', os.path.join(placeholders['PGHOME'], 'pgroot'))
+    placeholders.setdefault('WALE_TMPDIR', os.path.abspath(os.path.join(placeholders['PGROOT'], '../tmp')))
```

Comment:
It uses TMPDIR for keeping produced files there before uploading them to S3.
Total size of these files during backup could be quite big. Every file up to 1GB (lzo algorithm is fast but not very efficient). Number of parallel streams depends on amount of CPU cores.
But the weird thing is - sometimes something goes wrong (probably wal-e crashes) and temp files stays there forever...